helm_deploy/apply-for-legal-aid/templates/deployment.yaml

```diff
@@ -158,6 +158,11 @@ spec:
               secretKeyRef:
                 name: apply-for-legal-aid-elasticache-instance-output
                 key: auth_token
+          - name: TEST_LEVEL_NINO_VALIDATION
+            valueFrom:
+              secretKeyRef:
+                name: {{ template "apply-for-legal-aid-uat.fullname" . }}
```

Comment:
I think it should be `"apply-for-legal-aid.fullname"` instead of `"apply-for-legal-aid-uat.fullname"`